styles/components/_search.scss

```diff
@@ -232,7 +232,10 @@
     @include ktheme() {
       border-bottom-color: theme('placeholder-color');
       background: theme('background-color');
-      color: theme('text-color');
+      color: theme('text-color') !important;
+      &:hover {
+        background: theme('background-color') !important;
+      }
```

Comment:
```suggestion
      background: theme('background-color') !important;
      color: theme('text-color') !important;
```

Reply:
I guess it would do the same and have less dumplicate code.

Reply:
there is hover effect coming from styling of dropdowns in navbar which I'm overwriting here without changing the rest of the navbar. without this, the navbar hover effect gets applied here as well